src/manifest.ts

```diff
@@ -581,27 +587,34 @@ export class Manifest {
       }
     }
 
-    const needsBootstrap = releasesFound < expectedReleases;
-
     if (releasesFound < expectedReleases) {
-      this.logger.warn(
-        `Expected ${expectedReleases} releases, only found ${releasesFound}`
-      );
+      const errMessage = `Expected to find ${expectedReleases} releases, but ${
```

Comment:
I am still not sure about this. This will break customers and they most likely won't know what to do. Can we remedy it ourselves? If only one version is missing, we can maybe ignore it (and maybe generate a duplicate version)? Only when more than one is missing would we error out?

Reply:
Ok, let's keep this PR open for now, it's nothing urgent, we can change it later.